algorithms/C/maths/decimal_to_binary.c

```diff
@@ -0,0 +1,30 @@
+/* This program is used to convert decimal numbers to their binary equivalent
+   Time Complexity : O(logn)
+   Auxiliary Space: O(1)
+*/
+
+#include <stdio.h>
+void decimal_to_binary(int number)
+{
+	int i = 0;
+	int binary[32];
+
+	while(number > 0)
```

Comment:
What if the `number` is zero or less than zero ? Does your program handle that accordingly ?